texastribune/donations-app
# Update member form

#### What's this PR do?

Updates membership form to point to our 2016 form. Makes it point to the same url as our site so form in future only needs to be updated in one place.
#### Why are we doing this? How does it help us?

Request from Suzanne to update the form.
#### How should this be manually tested?

Visit the main page locally. At the bottom of the right column, click on the "Prefer to mail a form?" link. Verify it's an updated 2016 membership form PDF that doesn't include Mary's contact info anywhere.
#### Have automated tests been added?

no
#### Are there performance implications? If adding JS can it be done async? Do images/CSS/JS have cache headers set?

no
#### What are the relevant tickets?

Email request.
#### How should this change be communicated to end users?

Let Suzanne know this is updated.
#### Next steps?
#### Smells?
#### TODOs:
#### Has the relevant documentation/wiki been updated?
#### Technical debt note

Less since can switch out without deploying next time.
